miniwob/environment.py

```diff
@@ -0,0 +1,242 @@
+import logging
+import os
+import sys
+
+import gymnasium as gym
+
+from miniwob.action import MiniWoBActionSpace
+from miniwob.instance import MiniWoBInstance
+from miniwob.state import MiniWoBStateSpace
+
+
+class MiniWoBEnvironment(gym.Env):
+    """MiniWoB environment."""
+
+    metadata = {"render_modes": ["human"]}
```

Comment:
The "human" mode uses the normal ChromeDriver (showing the GUI to the user), while "None" uses the headless ChromeDriver. Neither of these require calling `render()`. I can add a comment to clarify this.

Reply:
Can we run the ChromeDriver "headless" by default or is the screen always shown?

Reply:
I made it so that it runs headless be default, and only shows the screen in "human" mode. Should I change that?

Reply:
Yes

Reply:
Sorry for not being clear. Just to confirm, should I change it so that the "human" render mode is default? Do I need to change to change anything else about the render modes?

Reply:
Added a comment regarding the render modes. Setting None ("headless") as default for now.